### DISCUSSION TITLE
Nameless "ghost" action blocking merge requests

### DISCUSSION THREAD

**Discussion by Pfeil** [community/community#77196] [Actions] [answered]
### Select Topic Area

Bug

### Body

There is an action that never gets finished (and also does not seem running). It does not have a name, no details button, and no action in the action tab (it seems). It probably is a leftover from an older CI config where we still tested against openjdk 11. There may be a bug or some misconfiguration, not sure. It has been there for a while, but got recently very persistent. It happens on every new PR. Until today, I just ignored it, hoping it would be solved over time. Example:

https://github.com/kit-data-manager/pit-service/pull/181

![grafik](https://github.com/community/community/assets/4805153/c8004efb-5b01-4878-a3f9-b678acd2c98f)

Note how the commit in the PR has a green tick.

I can not get any information about the pending action. It does not seem to be in the queue, not under any certain status like running or pending. The "11" indicates it is a leftover of an older CI config where we tested against openjdk 11. But this configuration does not exist anymore. It is possible to get rid of it (also for new PRs)? If there is no obvious solution, where can I report it?

**Comment by CheetahCodes21** [comment]
It seems that the pending action you're encountering is a result of an outdated CI configuration that is still lingering in your repository. The "11" in the action's title suggests that it's related to testing against OpenJDK 11, which is no longer part of your CI setup. This outdated configuration is causing the action to appear as pending, even though it's not actually running or producing any output.

To resolve this issue, you'll need to remove the outdated CI configuration from your repository. Here are the steps you can follow:

1. **Identify the outdated CI configuration:** Start by locating the outdated CI configuration file. This could be a YAML file or a script that defines the workflow for testing against OpenJDK 11.

2. **Remove the outdated CI configuration:** Once you've identified the outdated configuration file, remove it from your repository. This will prevent it from being triggered for new pull requests.

3. **Force re-run of the CI pipeline:** After removing the outdated configuration, force a re-run of the CI pipeline for the pull request in question. This will ensure that the CI pipeline runs with the updated configuration and that the pending action is no longer visible.

If the issue persists after removing the outdated configuration and forcing the CI pipeline to re-run, you can report it to the GitHub Actions team. They can provide further assistance in troubleshooting the issue and identifying any underlying bugs or misconfigurations.

Here's how to report the issue to GitHub Actions:

1. **Create a new issue in the GitHub Actions repository:** Go to the GitHub Actions repository and create a new issue.

2. **Provide a clear and concise description of the issue:** Explain the problem you're encountering with the pending action and include any relevant screenshots or error messages.

3. **Attach the relevant log files:** If there are any log files associated with the pending action, attach them to the issue report. This will help the GitHub Actions team investigate the issue more effectively.

4. **Submit the issue and wait for a response:** Once you've provided all the necessary information, submit the issue and wait for a response from the GitHub Actions team. They will review your issue and provide guidance on how to resolve the problem.

**Comment by Pfeil** [comment, marked as answer]
Never mind, I found the solution on the second page of the Google search results (desperation, I guess), here: https://github.com/orgs/community/discussions/26104
